base/src/main/java/com/smartdevicelink/streaming/video/VideoStreamingParameters.java

```diff
@@ -168,9 +178,12 @@ public void update(VideoStreamingParameters params) {
      */
     public void update(VideoStreamingCapability capability, String vehicleMake) {
         if (capability.getMaxBitrate() != null) {
-            this.bitrate = capability.getMaxBitrate() * 1000;
+            // Taking lower value as per SDL 0323 :
+            // https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0323-align-VideoStreamingParameter-with-capability.md
+            this.bitrate = Math.min(this.bitrate, capability.getMaxBitrate() * 1000);
```

Comment:
Hello @shiniwat, while testing I noticed an issue with taking the min bitrate. The sdl_hmi systemCapabilities that I was using had the max int value (2147483647), once we multiple by 1000 the int overflows and turns negative resulting in -1000 being the min value.
Since the max value for the bitRate per the [rpc_spec](https://github.com/smartdevicelink/rpc_spec/blob/8b8c996c738711c4235c8574c179354cbf8982e2/MOBILE_API.xml#L3278) is 2147483647 I think we should add a check to ensure the capabilities bitrate has not overflowed before updateing to the min value
```
if (capability.getMaxBitrate() * 1000 >= 0) {
    this.bitrate = Math.min(this.bitrate, capability.getMaxBitrate() * 1000);
}
```

Reply:
@RHenigan, this is good catch! I have updated as you suggested. Thanks.